.github/workflows/create_release_branch.yml

```diff
@@ -95,5 +97,8 @@ jobs:
                  \"draft\": true, \
                  \"body\":\":robot: **Beep boop I am a bot**\n\
                             This is an automatically created PR for version ${VERSION} to merge changes from ${RELEASE_BRANCH_NAME} to ${TARGET_BRANCH}.\n \
+                            Next steps for release process:\n \
+                            - Run CI to build Container images and CLI and Helm Charts\n \
+                            - Run Integration Tests to verify everything works and publish the release\n \
                             \"}" \
-                 https://api.github.com/repos/keptn/keptn/pulls || true
+                 https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls || true
```

Comment:
Is $GITHUB_REPOSITORY a global env?

Reply:
Yes. https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables